src/ui/SubtitleEdit.csproj

```diff
@@ -284,18 +284,4 @@
     <Compile Remove="Logic\VideoPlayers\DirectShowLib2005.cs" />
     <EmbeddedResource Remove="Forms\GetDictionaries.resx" />
   </ItemGroup>
-  <Target Name="BeforeCompile">
-    <XmlPeek XmlInputPath="$(ProjectDir)packages.config" Query="//package[@id='NHunspell']/@version">
-      <Output TaskParameter="Result" PropertyName="HunspellVersion" />
-    </XmlPeek>
-    <PropertyGroup>
```

Comment:
NHunspell is now nuget package (using PackageReference) so `..\..\packages\NHunspell.$(HunspellVersion)\content\` doesn't exist anymore